### DISCUSSION TITLE
KyuubiInstance always returns `0.0.0.0.10099` in ha mode on EKS.

### DISCUSSION THREAD

**Discussion by HaoYang670** [apache/kyuubi#5930] [Q&A]
I deployed Kyuubi on EKS in HA mode. When I call `createSession` to the ALB (I always clear the cookies so that the request can send to different kyuubi servers), the kyuubi Instances returned are always `0.0.0.0.10099`. Is it desirable? I assumed different kyuubi server should return different kyuubiInstance

**Comment by pan3793** [comment]
@HaoYang670 there are some related discussions at https://github.com/apache/kyuubi/issues/5010

**Reply by pan3793** [reply]
BTW, how do you deploy Kyuubi on EKS? using the latest helm chart may help, because of changing from Deployment to StatefulSet, see https://github.com/apache/kyuubi/pull/5062#issuecomment-1702589215

**Reply by HaoYang670** [reply]
Thanks @pan3793. The host is dynamic after commenting out `kyuubi.frontend.bind.host`, `front.rest.bind.host` and `Dkyuubi.frontend.bind.host`
